inst/app/modules/plots.R

```diff
@@ -0,0 +1,327 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# UI Side ####
+#~~~~~~~~~~~~~~~~~~~~~~~~
+
+inputsUI <- function(id){
+  ns <- NS(id)
+  tagList(
+    radioGroupButtons(ns("score"), "Choice of scoring", choices = c("All"= "all", "Coverage" = "cover", "Difference" = "diff", "Repeat" = "repeats", "Taper" = "taper", "Trend" = "trend"), individual = F, direction = "vertical", justified = F, width = "100%",
```

Comment:
two things: 

1. change the label from "Choice of scoring" to "PLANES Component(s)"
2. we need to be able to select multiple choices *except if "all" is chosen. in other words, as it's written now you can only select "all" or one component at a time. but a user may want to run something like just "cover" and "diff".